example/admission-webhook/deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: prometheus-operator-admission-webhook
+    app.kubernetes.io/version: 0.54.0
+  name: prometheus-operator-admission-webhook
+  namespace: default
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: prometheus-operator-admission-webhook
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: prometheus-operator-admission-webhook
+      labels:
+        app.kubernetes.io/name: prometheus-operator-admission-webhook
+        app.kubernetes.io/version: 0.54.0
+    spec:
+      automountServiceAccountToken: false
+      containers:
+      - image: quay.io/prometheus-operator/prometheus-operator:v0.54.0
+        name: prometheus-operator-admission-webhook
+        ports:
+        - containerPort: 8443
```

Comment:
isn't the default port 8080?

Reply:
it was but that changes in https://github.com/prometheus-operator/prometheus-operator/pull/4494/files#diff-fbea22ad8e57f3a3046a6dea17a28f1d3f2fa6476b57f0074318cf983030d83eR61

I've changed to 8443 as default since we default to enable TLS but I don't mind keeping 8080 if you prefer

Reply:
No this is fine for me!